__tests__/store/selectors/subjectsPicker.test.js

```diff
@@ -0,0 +1,109 @@
+import subjectsPicker from '../../../src/store/selectors/subjectsPicker'
+
+describe('subjectsPicker', () => {
+  it('should return object', () => {
```

Comment:
I do not think testing it in such detail is really necessary, but as you have this already done, then let's keep it.